docs: Improve log usability (#2951)
bonitasoft/bonita-doc: modules/setup-dev-environment/pages/logging.adoc

You can exploit the Mapped Diagnostic Context (MDC) information from the Bonita Runtime log to integrate it in an external Log Management System. +
These instructions demonstrate how to do it with https://eclipse.dev/tracecompass/[Eclipse Trace Compass] a free and open source traces and logs manager. You can adapt these to your favorite tool of choice.

[NOTE]
====

> This demonstration uses the textual log. In case you prefer using the JSON log format, you will find all the contextual information in `"contextMap"` attribute instead.

Reviewer: ```suggestion
This demo uses the textual log. If you prefer to use the JSON log format, you will find all the contextual information in the `"contextMap"` attribute instead.
```
